mindsdb/interfaces/storage/fs.py

```diff
@@ -92,10 +92,10 @@ def __init__(self):
 
     def get(self, local_name, base_dir):
```

Comment:
It is a common function, not used only by predict method
I would update predict method and instead line `self.model_storage.fileStorage.pull()`:
- use `self.model_storage.fileStorage.pull_path(update=False)` if it will work
- or update pull method and use `self.model_storage.fileStorage.pull(update=False)`

Reply:
I agree with Andrey, `fs.get` should be improved. But for now i think that can be merged as is